core/src/main.cpp

```diff
@@ -276,19 +285,8 @@ constexpr auto default_jsonschema() -> std::string_view
                                 "mode",
                                 "client_id",
                                 "redirect_uri",
-                                "tls_certificates_directory"
-                            ],
-                            "oneOf": [
-                                {{
-                                    "required": [
-                                        "irods_user_claim"
-                                    ]
-                                }},
-                                {{
-                                    "required": [
-                                        "user_attribute_mapping"
-                                    ]
-                                }}
+                                "tls_certificates_directory",
```

Comment:
does this have to be required... hmmm...  maybe it does.

Reply:
Relevant: #243